opensquirrel/squirrel_ir.py

```diff
@@ -114,7 +114,7 @@ def __eq__(self, other):
         return self.qubit == other.qubit and np.allclose(self.axis, other.axis, atol=ATOL)
 
     def accept(self, visitor: SquirrelIRVisitor):
```

Comment:
This should have a return type

Reply:
Yup. I still have to update my branch with the changes from Stan's PR (the type hinting one).